gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ format.version = "1.1"
 
 [versions]
 jackson = "2.18.1"
-edc-build = "0.10.1"
+edc-build = "0.11.0-SNAPSHOT"
```

Comment:
this will cause errors on next release, unfortunately edc-build plugin needs to stick to the latest proper release version (for the chicken-egg problem)

Reply:
🤦‍♂️ right...